packages/storybook-ui-components/stories/1-Badge.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+import { Badge } from "@cockroachlabs/ui-components";
```

Comment:
Ah, I see! So we will import components from other packages and display them in one storybook instance?

Reply:
Yes, that was my plan. To have the _Storybook_ be a separate package that will depend on UI Components to keep a separation of concerns.